### PR TITLE
sami is the best

### PR DESCRIPTION
**Pull request by samanthamorco**
This is my awesome code. Please accept.

### CODE REVIEW

**Comment by samanthamorco** [comment]
+1

File: app/views/pages/index.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+<h1>SAMI IS THE BEST ABSOLUTELY #1!!!!!!</h1>
```
Comment (samanthamorco) [inline app/views/pages/index.html.erb]: Sami is not the best. Wtf. Change this.

Comment (peterastrong) [reply app/views/pages/index.html.erb]: not sure about that

**Comment by samanthamorco** [comment]
+1

**Pull request merged**